src/Maker/MakeCrud.php

```diff
@@ -95,6 +101,15 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
 
     public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
     {
+        if (Kernel::VERSION_ID >= 50300) {
+            $this->enableHttpMethodOverride($generator);
+
+            $io->newLine();
+            $overrideMessage = 'Enabled framework.http_method_override - see <href=https://symfony.com/doc/5.3/reference/configuration/framework.html#http-method-override>https://symfony.com/doc/5.3/reference/configuration/framework.html#http-method-override</>';
+            $io->text($overrideMessage);
+            $io->newLine();
+        }
```

Comment:
You did a nice job with this. But I think the better solution is to stop relying on this setting. In other words, update `make:crud` to just send simple POST requests. I don't see any issue with this - I'm not sure it's very useful to imitate HTTP methods in a browser setting.

Reply:
done